src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/settingsComponents/GroupFeedback/GroupFeedbackRow.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import {
+  ActionRow, Form, Icon, IconButton, Col, Row,
+} from '@edx/paragon';
+import { DeleteOutline } from '@edx/paragon/icons';
+import PropTypes from 'prop-types';
+import messages from '../../messages';
+
+export const GroupFeedbackRow = ({
+  value,
+  handleAnswersSelectedChange,
+  handleFeedbackChange,
+  handleEmptyFeedback,
+  handleDelete,
+  answers,
+  // injected
+  intl,
+}) => (
+  <ActionRow className="mb-3">
+    <Col className="px-0">
```

Comment:
<img width="321" alt="Screenshot 2023-02-07 at 11 26 56 AM" src="https://user-images.githubusercontent.com/42981026/217303782-d898a78c-8fdd-44d2-96ca-28424fb2a6bd.png">
This is what the `GroupFeddbackRow` looks like for me. I suggest using an `ActionRow` to align the input field and delete icon.

Reply:
Ah, that might be a better approach. I was under the assumption from the mocks that the delete button had to be in the vertical middle of the answer checkboxes and input. Good catch.